src/webview/js/js.js

```diff
@@ -216,7 +216,7 @@ function midMessagePeer(top: number, bottom: number): ?Element {
   }
 
   // $FlowFixMe: doesn't know about Document#elementsFromPoint
-  const midElements: Array<Element> = document.elementsFromPoint(0, midY);
+  const midElements: Array<HTMLElement> = document.elementsFromPoint(0, midY);
```

Comment:
Should remove the fixme too, right?

... Ah, got it, this is all for a *future* version of Flow. :slightly_smiling_face: The commit message sounded like it was describing something already broken:

    flow: fix return type of `elementsFromPoint`
    
    `document.elementsFromPoint` was added in Flow v0.98.0. Bring our
    declared return type into compliance.
    
    https://github.com/facebook/flow/commit/c34fd59cf346650fbf2eeaedc38af9adf1d17da8

One thing I did a lot in the last round of this, which can help make this kind of story clearer, is to put the upcoming version in the future tense. So e.g. "will appear in Flow v0.98", or "Flow v0.98 will bring us a libdef for `document.elementsFromPoint`." (That Flow version already exists, but it hasn't yet appeared in our codebase, and will when we update to it.)